Collection alias ut fail

#### Steps/Code to reproduce:

https://github.com/milvus-io/milvus/pull/8215/checks?check_run_id=3671938039

#### Expected result:

Collection alias ut succeed.

#### Actual results:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x1bda792]

goroutine 890 [running]:
github.com/milvus-io/milvus/internal/distributed/rootcoord/client.(*GrpcClient).CreateAlias.func1(0xc00217afc0, 0x1b, 0xc003fc76c0, 0x1)
	/go/src/github.com/milvus-io/milvus/internal/distributed/rootcoord/client/client.go:565 +0xb2
github.com/milvus-io/milvus/internal/distributed/rootcoord/client.(*GrpcClient).recall(0xc00217afc0, 0xc0042ddb10, 0xc000000002, 0xc003f26910, 0xc0042ddbb0, 0x1a98633)
	/go/src/github.com/milvus-io/milvus/internal/distributed/rootcoord/client/client.go:212 +0x27d
github.com/milvus-io/milvus/internal/distributed/rootcoord/client.(*GrpcClient).CreateAlias(0xc00217afc0, 0x339ab20, 0xc0040063f0, 0xc003fc25a0, 0x0, 0x43138739af0beef9, 0x43138739af0beef9)
	/go/src/github.com/milvus-io/milvus/internal/distributed/rootcoord/client/client.go:564 +0xb3
github.com/milvus-io/milvus/internal/proxy.(*CreateAliasTask).Execute(0xc003fc6180, 0x339ab20, 0xc0040063f0, 0x1, 0x0)
	/go/src/github.com/milvus-io/milvus/internal/proxy/task.go:4702 +0xbd
github.com/milvus-io/milvus/internal/proxy.(*taskScheduler).processTask(0xc0026285a0, 0x33b1200, 0xc003fc6180, 0x33b0220, 0xc002634240)
	/go/src/github.com/milvus-io/milvus/internal/proxy/task_scheduler.go:470 +0x882
github.com/milvus-io/milvus/internal/proxy.(*taskScheduler).definitionLoop(0xc0026285a0)
	/go/src/github.com/milvus-io/milvus/internal/proxy/task_scheduler.go:489 +0x333
created by github.com/milvus-io/milvus/internal/proxy.(*taskScheduler).Start
	/go/src/github.com/milvus-io/milvus/internal/proxy/task_scheduler.go:856 +0x8b
FAIL	github.com/milvus-io/milvus/internal/proxy	92.584s
FAIL
Makefile:115: recipe for target 'test-go' failed
2
Error: Process completed with exit code 2.


#### Environment:
- Milvus version(e.g. v2.0.0-RC2 or 8b23a93):
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others: 

#### Configuration file:


#### Additional context:

--- a/internal/distributed/rootcoord/client/client.go
+++ b/internal/distributed/rootcoord/client/client.go
@@ -562,21 +562,45 @@ func (c *GrpcClient) GetMetrics(ctx context.Context, in *milvuspb.GetMetricsRequ
 
 func (c *GrpcClient) CreateAlias(ctx context.Context, req *milvuspb.CreateAliasRequest) (*commonpb.Status, error) {
 	ret, err := c.recall(func() (interface{}, error) {
-		return c.grpcClient.CreateAlias(ctx, req)
+		client, err := c.getGrpcClient()
+		if err != nil {
+			return nil, err
+		}
+
+		return client.CreateAlias(ctx, req)
 	})
+	if err != nil || ret == nil {
+		return nil, err
+	}
 	return ret.(*commonpb.Status), err
 }
 
 func (c *GrpcClient) DropAlias(ctx context.Context, req *milvuspb.DropAliasRequest) (*commonpb.Status, error) {
 	ret, err := c.recall(func() (interface{}, error) {
-		return c.grpcClient.DropAlias(ctx, req)
+		client, err := c.getGrpcClient()
+		if err != nil {
+			return nil, err
+		}
+
+		return client.DropAlias(ctx, req)
 	})
+	if err != nil || ret == nil {
+		return nil, err
+	}
 	return ret.(*commonpb.Status), err
 }
 
 func (c *GrpcClient) AlterAlias(ctx context.Context, req *milvuspb.AlterAliasRequest) (*commonpb.Status, error) {
 	ret, err := c.recall(func() (interface{}, error) {
-		return c.grpcClient.AlterAlias(ctx, req)
+		client, err := c.getGrpcClient()
+		if err != nil {
+			return nil, err
+		}
+
+		return client.AlterAlias(ctx, req)
 	})
+	if err != nil || ret == nil {
+		return nil, err
+	}
 	return ret.(*commonpb.Status), err
 }

